JASP-Engine/JASP/po/R-nl.po

```diff
@@ -6337,73 +6756,312 @@ msgstr "Posterior Kengetallen Voor Submodel %s"
-"provided."
+"Probability that Reliability Statistic is Larger than %.2f and Smaller than "
+"%.2f"
 msgstr ""
-"%1$i van de observaties zijn gebruikt, %2$i zijn uitgesloten %3$s, en %4$i "
-"zijn gegeven."
+
+msgid "Posterior Plots"
+msgstr "Posterior Grafieken"
+
+msgid "If Item Dropped Posterior Plots"
+msgstr "Als Item Wordt Weggelaten Posterior Grafiek"
+
+msgid "Item Dropped"
+msgstr "Item Wordt Weggelaten"
+
+msgid "Factors"
+msgstr "Factoren"
+
+msgid "Convergence Traceplot"
+msgstr "Convergentie Spoordiagram"
+
+msgid "Iterations"
+msgstr "Iteraties"
+
+msgid "%.2f"
+msgstr "%.2f"
+
+msgid "The analysis crashed with the following error message:"
+msgstr "De analyse is onderbroken met de volgende foutmelding:"
+
+msgid "Frequentist Scale Reliability Statistics"
+msgstr "Frequentistische Schaal Validiteitsstatistieken"
+
+msgid "Point estimate"
+msgstr "Puntschatting."
 
 msgid ""
-"%s\n"
-"Warning: Greatest lower bound can only be calculated for three or more "
-"variables."
+"%sMcDonald's ω estimation method switched to PFA because the CFA\n"
+"                                          did not find a solution."
 msgstr ""
-"%s\n"
-"Waarschuwing: Grootste ondergrens kan alleen worden berekend voor drie of "
-"meervariabelen."
+"%sMcDonald's ω schattingsmethode is omgeschakeld naar PFA omdat de CFA\n"
+"                                          geen oplossing vond."
 
 msgid "If item dropped"
 msgstr "Als item wordt weggelaten"
 
-msgid "item-rest correlation"
-msgstr "item-rest correlatie"
+msgid "Frequentist Individual Item Reliability Statistics"
+msgstr "Frequentistische Individuele Item Validiteitsstatistieken"
+
+msgid ""
+"McDonald's ω estimation method for item-dropped statistics\n"
+"                                    switched to PFA because the CFA did not "
+"find a solution."
+msgstr ""
+"%sMcDonald's ω schattingsmethode is omgeschakeld naar PFA omdat de CFA\n"
+"                                          geen oplossing vond."
+
+msgid "Fit Measures of Single Factor Model Fit"
+msgstr "Pasmaat van Enkele Factor Model passing"
+
+msgid ""
+"Fit measures cannot be displayed because the McDonald's ω \n"
+"                                               estimation method switched \n"
+"                                               to PFA as the CFA did not "
+"find a solution."
+msgstr ""
+"De passingmaat kan niet worden getoond omdat de schattingsmethode voor McDonald's ω \n"
+"veranderd is naar PFA, want de CFA leidde niet tot een oplossing."
+
+msgid "The priors for publication bias were set incorrectly."
+msgstr "De a priori verdeling voor publicatieonzuiverheid zijn verkeerd ingesteld."
+
+msgid "MCMC error"
+msgstr "MCMC fout"
+
+msgid "ESS"
+msgstr "ESS"
+
+msgid "%s is on %s scale."
+msgstr "s is op %s schaal."
+
+msgid "<em>p</em>-values interval %s"
+msgstr "<em>p</em>-waarden interval %s"
+
+msgid "The weights (%s) correspond to %s <em>p</em>-values."
```

Comment:
Maar zo te zien is dat op meerdere plekken niet gedaan...

Reply:
De posities? Ik heb geen posities gezien.

Reply:
%1$s en %2$s, maar dat had de codeschrijver al moeten doen. Ik heb @FBartos erop gewezen dat ie dat moet doen. En hij gaat er een issue voor maken voor na de release. Want voor nu maakt het niet uit, nederlands heeft zelfde volgorde als engels (grotendeels)